lib/std/json.zig

```diff
@@ -2731,6 +2731,9 @@ pub const StringifyOptions = struct {
 
             /// Should unicode characters be escaped in strings?
             escape_unicode: bool = false,
+
+            /// Should optional fields with null value be written?
+            write_null_optional_fields: bool = true,
```

Comment:
Why is this under `StringOutputOptions`?

Reply:
Good question. This was an error on my part